passage-2/README.md

```diff
@@ -0,0 +1,109 @@
+# Passage Relaunch Instructions 
+
+**Please note that this genesis file has been updated after taking a state export of `passage-1` at height 4088500. Validators who were validating on `passage-1` have to REUSE the signing key(priv_validator-key.json) to validate on `passage-2`**
```

Comment:
Shall we add an instruction to backup the key. So that it would be handy if they mess up